content/millennium/dstu2/infrastructure/document-reference.md

```diff
@@ -34,7 +34,10 @@ The following fields are returned if valued for clinical documents:
 * [Indexed date/time](http://hl7.org/fhir/DSTU2/documentreference-definitions.html#DocumentReference.indexed){:target="_blank"}
 * [Status (typically current)](http://hl7.org/fhir/DSTU2/documentreference-definitions.html#DocumentReference.status){:target="_blank"}
 * [Document status (typically final or amended)](http://hl7.org/fhir/DSTU2/documentreference-definitions.html#DocumentReference.docStatus){:target="_blank"}
-* [ContentType and URL (fully qualified link to the document](http://hl7.org/fhir/DSTU2/documentreference-definitions.html#DocumentReference.content.attachment){:target="_blank"}
+* [Document Attachment](https://hl7.org/fhir/dstu2/documentreference-definitions.html#DocumentReference.content.attachment){:target="_blank"}
+  * [Attachement ContentType](https://hl7.org/fhir/dstu2/datatypes-definitions.html#Attachment.contentType){:target="_blank"}
+  * [URL (fully qualified link to the document)](https://hl7.org/fhir/dstu2/datatypes-definitions.html#Attachment.url){:target="_blank"}
+  * [Title](https://hl7.org/fhir/dstu2/datatypes-definitions.html#Attachment.title){:target="_blank"}
```

Comment:
can we put this through a layer of content, then a layer of attachment, then a layer of contentType/url/title?

Reply:
[Layered](https://github.com/cerner/fhir.cerner.com/pull/337/commits/51acc559d2f17d86dad80ac82201628a4caaf31b#diff-b5f0beeb0c76f04f4fcbe824eb832abeR37-R41)